app/uk/gov/hmrc/eusubsidycompliancefrontend/forms/ClaimDateFormProvider.scala

```diff
@@ -78,7 +78,7 @@ case class ClaimDateFormProvider(timeProvider: TimeProvider) extends FormProvide
     case (d, m, y) =>
       localDateFromValues(d, m, y)
         .map { parsedDate =>
-          val today = timeProvider.today(ZoneId.of("Europe/London"))
+          val today = timeProvider.zonedToday(ZoneId.of("Europe/London"))
```

Comment:
Renamed today(zone: Zone) to  zonedToday as overloading is not really recommended in Scala as it affects the ability to pass methods as anonymous functions.